### PR TITLE
[raml2html] Initial plan and tests

### PR DESCRIPTION
**Pull request by MindNumbing**
Signed-off-by: Steven Marshall <SMarshall@chef.io>

### CODE REVIEW

File: controls/raml2html_works.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  impact 1.0
+  title 'Ensure raml2html binary is working as expected'
+  desc '
+  We first check that the raml2html binary we expect is present and then run version checks on both to verify that it is excecutable.
+  '
+
+  hab_pkg_path = command("hab pkg path #{plan_ident}")
+  describe hab_pkg_path do
+    its('exit_status') { should eq 0 }
+    its('stdout') { should_not be_empty }
```
Comment (gavindidrichsen) [inline controls/raml2html_works.rb:20]: Hi @MindNumbing.  What about an extra stderr check here:  its('stderr') { should be_empty }

**Review by gavindidrichsen — COMMENTED** [review]
Hi @MindNumbing.   One slight change, otherwise looks good

File: controls/raml2html_works.rb (hunk 1 above)
Comment (MindNumbing) [reply controls/raml2html_works.rb:20]: 👍

**Review by gavindidrichsen — APPROVED** [review]
LGTM

**Pull request merged**